### ISSUE TITLE
Parent <-- document --> child relationship

### ISSUE THREAD

**Issue by ginold** [c2corg/v6_ui#326]
Associating new documents is easy : I define which one is the parent and which one is the child, and I know the possible valid associations.
But the other way is not that simple. Sometimes I want to dissociate two documents and I obtain an error 'association does not exist' because I have no way of knowing which one is the parent and which one the child. Or do I ? If so, please tell me how I can get to know this relationship.

If it's not the case, maybe one of the possible solutions would be to send two IDs like `dissociateDocuments(ID1, ID2)` and the API will try one way or the other, if the 1st one does not pass (we could do the same, calling a 2nd time the dissociate method if the 1st time is unsuccessful).
Or maybe you have any better ideas?

Sooner or later we would need to have this information UI-side in order to determine the parent and it's children and to show it in the interface accordingly.

**Comment by tsauerwein** [comment]
> If it's not the case, maybe one of the possible solutions would be to send two IDs like dissociateDocuments(ID1, ID2) and the API will try one way or the other, if the 1st one does not pass

Yes, that doesn't sound too bad. Especially since you can not have two associations D1->D2 and D2->D1 at the same time. See https://github.com/c2corg/v6_api/issues/263

**Comment by tsauerwein** [comment]
Done in https://github.com/c2corg/v6_api/pull/264

**Comment by ginold** [comment]
fixed in #328

**Comment by asaunier** [comment]
> fixed in #328

Reopening the issue since the PR is not merged yet (as a result the issue is still valid).
